src/snowflake/cli/api/cli_global_context.py

```diff
@@ -166,6 +182,8 @@ def _collect_not_empty_connection_attributes(self):
             "schema": self.schema,
             "role": self.role,
             "warehouse": self.warehouse,
+            "session_token": self.session_token,
```

Comment:
do we need both to be present for this to work? If so, do we check anywhere if they are both present or both absent? (i.e. should we error out if user provides master-token without providing session-token?)

Reply:
You are right about this: these tokens must be used together. I will add appropriate error messages, thank you!

Reply:
I have added three error messages (with their corresponding unit tests). Please let me know if you think the messages can be improved.

1. For the case in which a session token or master token has been provided but the user has not specified that the connection is temporary
2. For the case in which only the session token was provided
3. For the case in which only the master token was provided
![image](https://github.com/snowflakedb/snowflake-cli/assets/127364992/b4732d6b-b35c-44ae-861c-a09e66e1e645)
![image](https://github.com/snowflakedb/snowflake-cli/assets/127364992/f3becc74-a52c-4ddc-8646-94a6b655a611)
![image](https://github.com/snowflakedb/snowflake-cli/assets/127364992/2ffddc01-f8cc-428b-adbe-b36a7e03485f)